pkgs/development/python-modules/django-rapyd-modernauth/default.nix

```
@@ -0,0 +1,42 @@
+{
+    lib, buildPythonPackage, fetchPypi,
+
+    django,
+    python-dotenv,
+    pytz,
+    setuptools,
+    sqlparse,
+    zipp
+}:
+
+buildPythonPackage rec {
+    pname = "django-rapyd-modernauth";
+    version = "0.0.4";
```

```suggestion
  pname = "django-rapyd-modernauth";
  version = "0.0.4";
```
we use 2 spaces for nix files